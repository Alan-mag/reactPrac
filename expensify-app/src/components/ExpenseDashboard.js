import React from "react";

const ExpenseDashboard = () => (
    <div>
        This is a dashboard component.
    </div>
);

export default ExpenseDashboard;