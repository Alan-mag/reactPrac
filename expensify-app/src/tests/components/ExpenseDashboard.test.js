import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboard';

test('should render Expense Dashboard component correctly', () => {
  const wrapper = shallow(<ExpenseDashboard />);
  expect(wrapper).toMatchSnapshot();
});