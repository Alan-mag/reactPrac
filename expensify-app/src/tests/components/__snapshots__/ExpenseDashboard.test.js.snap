// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Dashboard component correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
